xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-ui/src/XWiki/Lightbox/LightboxAdministration.xml

```diff
@@ -0,0 +1,332 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<xwikidoc version="1.4" reference="XWiki.Lightbox.LightboxAdministration" locale="">
+  <web>XWiki.Lightbox</web>
+  <name>LightboxAdministration</name>
+  <language/>
+  <defaultLanguage>en</defaultLanguage>
+  <translation>0</translation>
+  <creator>XWiki.Admin</creator>
+  <creationDate>1639138843000</creationDate>
+  <parent>WebHome</parent>
+  <author>XWiki.Admin</author>
+  <contentAuthor>XWiki.Admin</contentAuthor>
+  <date>1640076427000</date>
+  <contentUpdateDate>1639244117000</contentUpdateDate>
+  <version>5.6</version>
+  <title>LightboxAdministration</title>
+  <comment/>
+  <minorEdit>true</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>false</hidden>
+  <content>{{velocity output="false"}}
+#macro (stripHTMLMacro $displayOutput)
+  $stringtool.removeEnd($stringtool.removeStart($displayOutput, '{{html clean="false" wiki="false"}}'), '{{/html}}')
+#end
+#macro (showProperty $propertyName)
+  &lt;dl>
```

Comment:
I find it surprising that the `>` are not escaped to `&gt;`.

Reply:
In XML only `<` needs to be escaped.

Reply:
I agree but, using `mvn xar:format`, `>` is escaped too.

Reply:
I applied `mvn xar:format` twice, but it did not changed anything. But I found https://jira.xwiki.org/browse/XWIKI-10342 and https://dev.xwiki.org/xwiki/bin/view/IRC/xwikiArchive20140206 that discussed this problem in the past